bip-update-process.md

```diff
@@ -0,0 +1,706 @@
+recommend adoption, implementation, or deployment of the BIP. Where applicable, the authors must ensure that any
+proposed specification is solid, not unduly complicated, and definitive. Subsequently, the BIP’s content should only be
+adjusted in minor details, e.g., to improve language, clarify ambiguities, backfill omissions in the specification, add
+test vectors for edge cases, or address other issues discovered as the BIP is being adopted.
+
+A Complete BIP can only move to Deployed or Closed. Any necessary changes to the specification should be minimal and
+interfere as little as possible with ongoing adoption. If a Complete BIP is found to need substantial functional
+changes, it may be preferable to move it to Closed[^new-BIP], and to start a new BIP with the changes instead.
+Otherwise, it could cause confusion as to what being compliant with the BIP means.
+
+Complete is the final state for most successful Informational BIPs.
+
+#### Deployed
+
+A settled[^settled] BIP may be advanced to Deployed upon request by any community member with evidence that the idea
+described in the BIP is in active use. Such evidence includes for example: at least one project having deployed support
+for the BIP in mainnet software releases, a soft fork proposal’s activation criteria having been met on the network, or
+rough consensus for the BIP having been demonstrated.
+
+At that point, the BIP should be considered final and any breaking changes to the BIP should be proposed as a new
+separate BIP.[^new-BIP]
+
+##### Process BIPs
+
+A Process BIP may change status from Complete to Deployed when it achieves rough consensus on the mailing list. Such a
+proposal is said to have rough consensus if it has been open to discussion on the development mailing list for at least
+one month, and no person maintains any unaddressed substantiated objections to it. Addressed or obstructive objections
+may be ignored/overruled by general agreement that they have been sufficiently addressed, but clear reasoning must be
+given in such circumstances. Deployed Process BIPs may be modified indefinitely as long as a proposed modification has
+rough consensus per the same criteria.[^living-documents]
+
+#### Closed[^closed]
+
+A BIP that is _not in active use_, may be labeled Closed when its authors have stopped working on it, no longer
+recommend the proposed approach, or no longer pursue adoption of their Complete proposal. The reason for moving the
+proposal to Closed should be recorded in the Changelog section in the same commit that updates the status.
+
+##### Draft ↦ Closed
+
+BIP authors may decide on their own to change their BIP’s status from Draft to Closed. If a Draft BIP stops making
+progress, sees accumulated feedback unaddressed, or otherwise appears stalled for a year, the community may move the BIP
+to Closed unless the authors assert that they intend to continue work when contacted.
```

Comment:
It may be a good idea to provide a time frame here for considering when authors are non-responsive or no longer reachable.

Reply:
>  It may be a good idea to provide a time frame here for considering when authors are non-responsive or no longer reachable.

Good idea. By committing to a specific timeout here, authors can be informed upfront as to when their efforts will be considered abandoned. This could serve to encourage them to indicate when they will be unreachable, too.